test/array.cpp

```diff
@@ -409,13 +409,24 @@ TEST(array_reshape_3d) {
 }
 
 TEST(array_negative_strides) {
+  // Test with a negative stride in y.
   array_of_rank<int, 2> a({{0, 10, 3}, {0, 3, -1}});
-  ASSERT_EQ(a.x().stride(), 3);
   for_all_indices(a.shape(), [&](int x, int y) { a(x, y) = y; });
 
-  dense_array<int, 2> b = make_compact_copy(a);
-  for_each_index(b.shape(),
-      [&](const dense_array<int, 2>::index_type& i) { ASSERT_EQ(b(i), std::get<1>(i)); });
+  dense_array<int, 2> a_compact = make_compact_copy(a);
+  ASSERT(a_compact.y().stride() > 0);
```

Comment:
ASSERT_GT

Reply:
We only have ASSERT_LT :) This is just my local test.h...